server/modules/slack/slackIntegration.js

```diff
@@ -209,13 +211,9 @@ export const accessChannel = async (email, channelId, context) => {
       statusMessage: `${email} ${contextObject[context].message} channel`,
     };
   } catch (error) {
-    return {
-      slackUserId: null,
-      channelId,
-      channelName: channelInfo && channelInfo.channel.name,
-      type: context,
-      status: 'failure',
-      statusMessage: `${error.message}`,
-    };
+    if (`${error.message}` === 'An API error occurred: not_in_channel' || 'An API error occurred: already_in_channel' || 'An API error occurred: not_in_group') {
```

Comment:
We could do this like this:
```
const skipped = [
   'An API error occurred: not_in_channel',
   'An API error occurred: already_in_channel',
   'An API error occurred: not_in_group'
]
if (skipped.includes(error.message)) return responseObject(channelId, channelInfo, error, 'success');
return responseObject(channelId, channelInfo, error, 'failure');

Reply:
With that, when new error message is to be skipped, we just add it to be skipped array

Reply:
I think the @nzediegwu1 is sharing is ok and I would kindly suggest you consider it @zaabu

Reply:
Thanks @nzediegwu1  and @katunold . I am implementing the feedback.